src/mcts/search.cc

```diff
@@ -767,6 +767,16 @@ void Search::Wait() {
 Search::~Search() {
   Abort();
   Wait();
+  {
+    SharedMutex::Lock lock(nodes_mutex_);
+    for (auto& entry : shared_collisions_) {
```

Comment:
I was going to move this logic to a separate method, just forgot to do it earlier.

Reply:
Fixed.